flink/src/main/java/org/apache/iceberg/flink/actions/RewriteDataFilesAction.java

```diff
@@ -61,8 +61,9 @@ protected RewriteDataFilesAction self() {
     return this;
   }
 
-  public void maxParallelism(int parallelism) {
+  public RewriteDataFilesAction maxParallelism(int parallelism) {
```

Comment:
Would it make sense to also add `setMaxParallelism` in addition to this to match the Flink API?

Reply:
I think it's OK to use `maxParallelism(..)`  because the base class `BaseRewriteDataFilesAction` also have the setter method without the prefix `set`.